modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/layouts/1_dashboard/page-definition.json

```
@@ -61,21 +64,42 @@
 										"pageElements": [
 											{
 												"definition": {
-													"widgetInstance": {
-														"widgetConfig": {
-															"properties": "route=notification-sidebar"
-														},
-														"widgetName": "[$CLIENT_EXTENSION_ENTRY_ID:RAYLIFEAGENTPORTAL$]"
-													}
+													"fragment": {
+														"key": "avatar-dropdown-menu",
+														"siteKey": "AP1"
+													},
+													"fragmentConfig": {
+													},
+													"fragmentFields": [
+													],
+													"indexed": true
 												},
-												"type": "Widget"
+												"type": "Fragment"
 											}
 										],
 										"type": "Section"
 									}
 								],
 								"type": "FragmentDropZone"
 							},
+							{
+								"definition": {
+								},
+								"pageElements": [
+									{
+										"definition": {
+											"widgetInstance": {
+												"widgetConfig": {
+													"properties": "route=notification-sidebar"
+												},
+												"widgetName": "com_liferay_client_extension_web_internal_portlet_ClientExtensionEntryPortlet_20096_RAYLIFEAGENTPORTAL001"
```

Please, replace all the widget names in all `page-definitions.json`.

```suggestion
												"widgetName": "[$CLIENT_EXTENSION_ENTRY_ID:RAYLIFEAGENTPORTAL$]"
```